vllm/entrypoints/openai/serving_chat.py

```diff
@@ -100,9 +99,17 @@ def _parse_chat_message_content(
         return [ConversationMessage(role=role, content="\n".join(texts))], []
 
     async def create_chat_completion(
-        self, request: ChatCompletionRequest, raw_request: Request
+        self,
```

Comment:
I do not think we should change the signature of this function in the PR

Reply:
Sure, are you just trying to keep this method signature backwards compatible?

2 possibilities 
1) Make `raw_request: Optional[Request]` and default to never aborting if `raw_request is None`
2) I can make a fake `RawRequest` in the `run_batch.py` to pass in.

I'm fine either way, or if you had something else in mind. Let me know what you prefer.